src/lib/accessibility/Behaviors/Popup/popupFocusTrapBehavior.ts

```diff
@@ -0,0 +1,17 @@
+import { Accessibility } from '../../types'
+import popupBehavior from './popupBehavior'
+import * as _ from 'lodash'
+
+/**
+ * @description
+ * Adds role='button' to 'trigger' component's part, if it is not focusable element and no role attribute provided.
+ * Adds tabIndex='0' to 'trigger' component's part, if it is not tabbable element and no tabIndex attribute provided.
+ * Adds attribute 'aria-disabled=true' to 'trigger' component's part based on the property 'disabled'.
+ * Wraps component in FocusTrapZone, with default prop 'isClickableOutsideFocusTrap' set to 'true'.
```

Comment:
this is very confusing to the client. While client is aware of `aria-` and other well-known accessibility aspects, `FocusTrapZone` is an implementation detail. I would suggest to either
- introduce other general description
- omit this statement for now